code/modules/food_and_drinks/recipes/tablecraft/recipes_moth.dm

```diff
@@ -484,3 +484,15 @@
 	)
 	result = /obj/item/food/soup/red_porridge
 	subcategory = CAT_MOTH
+
+/datum/crafting_recipe/food/chili_sin_carne
```

Comment:
Should this be a moth recipe though ?

Reply:
I put it in the mothic section because its an ingredient for loaded curds but I can change it if thats needed